features/get_task.feature

```diff
@@ -2,4 +2,7 @@ Feature: Get a task
 
   Scenario: User gets a task
     TODO: Add steps for getting a task
-    
\ No newline at end of file
+  Given the user is using the task manager application for looking at or getting a task by name
+  And at least one task is available in the list 
```

Comment:
And a task with name "My Task" and description "My Description" is managed by the task manager app